sleap/gui/app.py

```diff
@@ -746,6 +746,22 @@ def new_instance_menu_action():
 
         labelMenu.addSeparator()
 
+        add_menu_item(
+            labelMenu,
+            "Extract clip and labels",
+            "Extract Clip and Labels...",
+            self.commands.exportClipVideo,
+        )
+
+        add_menu_item(
+            labelMenu,
+            "extract clip labels package",
+            "Extract Clip Labels Package...",
+            self.commands.exportClipPkg,
+        )
```

Comment:
Lets enable/disable these based on whether a clip is selected in `MainWindow._update_gui_state` (which is called periodically while the GUI is open). For example:
https://github.com/talmolab/sleap/blob/1eff33dbd9e342142092d0581f79e21560ecdf2c/sleap/gui/app.py#L1129

Reply:
Great suggestion! I have now changed these menu items to be enabled/disabled based on whether a clip is selected.